website/files/models/base.py

```diff
@@ -398,9 +399,10 @@ def copy_under(self, destination_parent, name=None):
 
     def move_under(self, destination_parent, name=None):
         self.name = name or self.name
+        if self.parent.node != destination_parent.node:
+            project_signals.file_moved_node.send(self, destination_node=destination_parent.node)
```

Comment:
I'm not sure what `name` is here, but it might be useful to pass to the signal. That said , I don't think it's necessary at this point.